scripts/composables/use_wallet.js

```diff
@@ -81,15 +90,17 @@ export const useWallet = defineStore('wallet', () => {
             } else {
                 await wallet.sign(tx);
             }
-            const res = await network.sendTransaction(tx.serialize());
-            if (res) {
-                await wallet.addTransaction(tx);
-            } else {
-                wallet.discardTransaction(tx);
-            }
-            return res;
+            return await sendTransaction(network, tx);
         }
     );
+    const createAuthoshieldTransactions = async (network, address, value) => {
+        const txs = await wallet.createAutoshieldTransactions(address, value);
+        let res = true;
+        for (const tx of txs) {
+            res = res && (await sendTransaction(network, tx));
```

Comment:
if the first sendTransaction fails why you still try with other txs?

Reply:
It doesn't, if `res` is `false` it doesn't evaluate `sendTransaction`